config/default.yaml

```diff
@@ -2,6 +2,10 @@ server:
   port: 5999
   appRoot: "/bucky"
 
+handlers:
```

Comment:
It's a little unclear to me why this belongs in a new section called `handlers`.  Why not put it in the general `server` section above?